lib/triggers/redditTrigger.js

```diff
@@ -50,11 +50,11 @@ RedditTrigger.prototype._respond = function(toId, message) {
 		var fullurl = that.options.redditapiurl + (!isNaN(steamid) ? "steamid/" + steamid : "reddit/" + steamid.replace("/u/","").replace("/user/","").replace("u/","").replace("user/",""));
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969986140/issues/source?bid=1980036&fileBranchId=2848073#l50)